openmmtools/integrators.py

```diff
@@ -1572,20 +1791,14 @@ def add_integrator_steps(self, splitting, measure_shadow_work, measure_heat, ORV
         self.addComputeGlobal("protocol_work", "protocol_work + (perturbed_pe - unperturbed_pe)")
 
         # Computing context updates, such as from the barostat, _after_ computing protocol work.
-        super(ExternalPerturbationLangevinIntegrator, self).add_integrator_steps(splitting, measure_shadow_work, measure_heat, ORV_counts, force_group_nV, mts)
-        self.addComputeGlobal("unperturbed_pe", "energy")
+        super(ExternalPerturbationLangevinIntegrator, self)._add_integrator_steps()
 
+        # Store final energy before perturbation
+        self.addComputeGlobal("unperturbed_pe", "energy")
 
 class VVVRIntegrator(LangevinIntegrator):
     """Create a velocity Verlet with velocity randomization (VVVR) integrator."""
-    def __init__(self,
-                 temperature=298.0 * simtk.unit.kelvin,
-                 collision_rate=91.0 / simtk.unit.picoseconds,
-                 timestep=1.0 * simtk.unit.femtoseconds,
-                 constraint_tolerance=1e-8,
-                 measure_shadow_work=False,
-                 measure_heat=True,
-                 ):
+    def __init__(self, *args, **kwargs):
```

Comment:
Ahh, thanks for fixing this! I had introduced code duplication in these subclasses by hard-coding the default arguments, instead of using `*args, **kwargs`